oap-server/server-bootstrap/src/main/resources/ui-initialized-templates/self-observability.yml

```diff
@@ -184,6 +184,42 @@ templates:
                   "metricLabels": "50,70,90,99",
                   "labelsIndex": "50,70,90,99",
                   "unit": "Millisecond"
+                },
+                {
+                  "width": 3,
+                  "title": "Thread Count",
+                  "height": "200",
+                  "entityType": "ServiceInstance",
+                  "independentSelector": false,
+                  "metricType": "REGULAR_VALUE",
+                  "metricName": "meter_oap_instance_jvm_thread_live_count,meter_oap_instance_jvm_thread_daemon_count,meter_oap_instance_jvm_thread_peak_count",
+                  "queryMetricType": "readMetricsValues",
+                  "chartType": "ChartLine",
+                  "unit": "Per Minute"
```

Comment:
what does the unit `Per Minute`  mean?  If you want to get the time duration metric you should use like `rate('PT1M')`  or `increase('PT1M')` in the expressions.

Reply:
done